src/scenes/Wallet/components/Balances/index.jsx

```diff
@@ -70,6 +78,11 @@ class MyBalances extends React.PureComponent {
       <Paper className={classes.myBalancePaper}>
         <Grid container spacing={0} className={classes.myBalanceGridContainer}>
           <Typography variant="title" className={classes.myBalanceTitle}>
+            <div className={classes.importantNoteContainer}>
```

Comment:
I would also say add the `injectIntl` in `ImportantNote`, but I know how much they don't want refactoring right now so we can hold off on that for later unless @dwalintukan says it's cool. It's only in 3 places in the codebase

Reply:
let's keep it the way it is right now. for certain important note messages, you would have to pass in the string to translate as well as some vars that go into that string.